examples/patchbuilder/patch-template-builder.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: bundle.gke.io/v1alpha1
+kind: PatchTemplateBuilder
+buildSchema:
+  required:
+    - K8sVersion
+  properties:
+    k8sVersion:
+      description: The kubernetes version for this build. This will be used as the image tag for Kubernetes images.
+      type: string
+targetSchema:
+  properties:
+    Registry:
+      description: The registry from which the image should be pulled. Should include
+        the trailing /.
```

Comment:
why include the trailing /?

Reply:
So you can leave it bare if you want. But since I ended up putting the default in here, I don't think you can do that anyway. So I will remove it.